TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/cluster.py

```diff
@@ -304,3 +305,39 @@ def __init__(self):
 class NoTargetClusterDefinedError(Exception):
     def __init__(self):
         super().__init__("Unable to continue without a target cluster specified")
+
+
+class AuthBase(BaseModel):
+    type: str
+
+
+class NoAuth(AuthBase):
+    type: str = "no_auth"
+
+
+class BasicAuth(AuthBase):
+    type: str = "basic_auth"
+    username: str
+    password: str
```

Comment:
Following up on our discussion I'll remove this type from the backend and frontend to avoid dealing with secure resources.

Reply:
👍